src/components/StudyCard/StudyCard.js

```diff
@@ -0,0 +1,49 @@
+import React, {useState} from "react"
+import classes from './StudyCard.module.css'
+
+const StudyCard = ({cardState}) => {
+    //  Different layout depending on image attribute
+    const [image, changeLayout] = useState(true)
+
+    const renderImageLayout = () => {
+        return cardState ?
+            <div>
+                <div className={classes.image_question}>
+                    <img className={classes.image_question_column + ' ' + classes.image}
+                         src="https://media.nature.com/lw800/magazine-assets/d41586-020-01430-5/d41586-020-01430-5_17977552.jpg" alt="card_image"/>
+                    <h1 className={ classes.wordFront + ' ' + classes.image_question_column}> What is the name of this animal? </h1>
+                </div>
+                <div className={classes.separator}/>
+                <h1 className={classes.wordFront}> 犬 </h1>
+            </div>
+            :
+            <div className={classes.image_question}>
+                <img className={classes.image_question_column + ' ' + classes.image}
+                     src="https://media.nature.com/lw800/magazine-assets/d41586-020-01430-5/d41586-020-01430-5_17977552.jpg" alt="card_image"/>
+                <h1 className={classes.wordFront + ' ' + classes.image_question_column}> What is the name of this animal? </h1>
+            </div>
```

Comment:
It is working properly but the second block is kinda redundant and React has to rerender the same image and text when the state is changed. So, I think it might be better to change the state like that.
```suggestion
                {cardState && (
                    <>
                        <div className={classes.separator} />
                        <h1 className={classes.wordFront}> 犬 </h1>{' '}
                    </>
                )}
             </div>
```